api/HTMLObjectElement.json

```
@@ -361,19 +361,19 @@
               "version_added": true
             },
             "opera": {
-              "version_added": true
+              "version_added": "40"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "40"
```

Opera Android doesn't actually have a version 40 as we've found.  Let's bump this up to the next available one, and this PR will be ready!

```suggestion
              "version_added": "41"
```